src/modules/retransmitter/retransmitter.prx

```diff
@@ -50,6 +50,18 @@
                     <priority>30</priority>
                     <stack_size>4096</stack_size>
                 </task>
+                <task>
+                    <name>task_read_to_buffer</name>
+                    <function>task_read_to_buffer_fn</function>
+                    <priority>32</priority>
+                    <stack_size>4096</stack_size>
+                </task>
+                <task>
+                    <name>task_publish_buffer</name>
+                    <function>task_publish_buffer_fn</function>
+                    <priority>31</priority>
```

Comment:
This task will never run because you never yield from the one above. You could maybe have the one above yield whilst waiting for a uart interrupt? (Uart interrupt handler is in boilerplate.c)

Reply:
Will do so on Monday night

Reply:
Done. Sorry for the delay